### ISSUE TITLE
Accessiblity support (like caniuse)

### ISSUE THREAD

**Issue by una** [una/YouMightNotNeedJS#49]
1-3 rating scale:

1 = no accessibility support
2 = partial support
3 = full support

for each demo

**Comment by ryanflorence** [comment]
Any reason to even have demos that can't reach 3 without JS? If you can't reach 3 without JS, then you absolutely need JS.

**Comment by una** [comment]
Yeah, i think its still a good demonstration of the powers of CSS (the point of this project is a showcase of "you can do things without JS that you may not have realized", which allow developers to be inspired by/build off of and learn new concepts. I think this is good for experimentation and learning purposes. It also doesn't mean there isn't a way for it to be fully a11y-compliant necessarily, but it means that **we** haven't found a way to make it a 3 yet with current browser technologies, which change. I think would counteract the spirit of the project, since the site is not a demo of accessible UI patterns, but of the power of CSS.

There's always been an accessibility warning on the site, this would make it more explicit and thus an improvement providing a warning about use in production.

**Comment by marcysutton** [comment]
I like the idea; it definitely hinges on the "you MIGHT" need JS part. And more information is better than none.

That said, part of me is afraid people will take even more accessibility shortcuts than they already do. But I think my fears about that have already been expressed elsewhere :)

**Comment by shadeed** [comment]
Nice idea! 

I fiddled with the below mockups:

![youmightnotneedjs-1](https://cloud.githubusercontent.com/assets/8038659/19335617/59dfcaf2-910d-11e6-94ea-92e93feb488a.jpg)

![youmightnotneedjs-2](https://cloud.githubusercontent.com/assets/8038659/19335621/614ee17e-910d-11e6-982c-0767148a7498.jpg)

Thoughts please? :)

**Comment by una** [comment]
Nice @shadeed :) I was thinking it could also open up with more info

![a11y1](https://cloud.githubusercontent.com/assets/1693164/19335836/68060292-90d4-11e6-9ad3-5e35bfa71812.png)
![a11y1_open](https://cloud.githubusercontent.com/assets/1693164/19335838/69884580-90d4-11e6-8176-3c4cef39bfce.png)

**Comment by una** [comment]
Also a thought, maybe accessibility support should be written out:
1. `Accessibility Support: none`
2. `Accessibility Support: partial`
3. `Accessibility Support: compliant`

With links to relevant content or an explanation of why it can't be accessible
